survey/src/main/resources/application.properties

```diff
@@ -1,9 +1,9 @@
 spring.application.name=survey-service
 
-spring.datasource.url=jdbc:postgresql://localhost:5432/survey_db
+spring.datasource.url=${SPRING_DATASOURCE_URL}
```

Comment:
Aren't we able to do:  `spring.datasource.url=jdbc:postgresql://${POSTGRES_HOST:-db}:${POSTGRES_PORT:-5432}/${POSTGRES_NAME}` and in docker-compose pass POSTGRES-related environments? It would be more consistent with what we have in processing already.

Reply:
SPRING_DATASOURCE_URL this is env specyfic for this service (look on docker compose) it mapped to the same instance but allows us to disconnect db in future (iteration4)